<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AirportEditPage</class>
 <widget class="QWidget" name="AirportEditPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" rowstretch="0,0,0,0,1">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>10</number>
   </property>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,1">
     <property name="spacing">
      <number>15</number>
     </property>
     <item row="1" column="1">
      <widget class="QLineEdit" name="enName_inp">
       <property name="maxLength">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="enUpdate_btn">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="newEnName_lbl">
       <property name="text">
        <string>Новое английское название:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QWidget" name="Z2" native="true"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="newRuName_lbl">
       <property name="text">
        <string>Новое русское название:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ruName_inp">
       <property name="maxLength">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="ruUpdate_btn">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
     <item>
      <widget class="QWidget" name="Z5" native="true"/>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="remove_btn">
       <property name="text">
        <string>Удалить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="Z4" native="true"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_3" columnstretch="0,0,1">
     <property name="bottomMargin">
      <number>20</number>
     </property>
     <property name="spacing">
      <number>15</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="enName_lbl">
       <property name="text">
        <string>Английское название:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="ruName_lbl">
       <property name="text">
        <string>Русское название:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QWidget" name="Z1" native="true"/>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="enNamePlace">
       <property name="text">
        <string>...........</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="ruNamePlace">
       <property name="text">
        <string>...........</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="mainInfo_lbl">
     <property name="text">
      <string>АЭРОПОРТ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>РЕДАКТИРОВАНИЕ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ruName_inp</tabstop>
  <tabstop>ruUpdate_btn</tabstop>
  <tabstop>enName_inp</tabstop>
  <tabstop>enUpdate_btn</tabstop>
  <tabstop>remove_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
